app/src/main/res/values/strings.xml

```diff
@@ -4,15 +4,15 @@
 
     <!-- Login activity -->
     <string name="app_title">beeline</string>
-    <string name="tag_line">Direct path to highest mutual attraction.</string>
+    <string name="tag_line">Your direct path to highest mutual attraction.</string>
     <string name="login_button_label">Log in with Facebook</string>
     <string name="background_content_description">Background texture</string>
 
     <!-- Introduction deck activity -->
     <string name="getting_started_button">Get started</string>
     <string name="rate_users_title">Pick the one you like better</string>
     <string name="rate_users_body">
-        Let the algorithm learn the nuanced landscape of your dating preferences versus the general yes/no buckets of swiping.
+        Let our algorithm determine the nuanced landscape of your dating preferences.
```

Comment:
Simplification is a good thing.

The wording is very old.

Remember, the comment that I made on a Google doc: A Beeline differentiator is that there is no algorithm that decides whom the user matches with. It's entirely decided by the users. The matchmaker called it with excitement: "You are the algorithm." So, the copy shouldn't say "our algorithm."

(Technically, every program is an algorithm. In common language, algorithms are meant to be complex logic that make allegedly intelligent decisions. The app doesn't make matching decisions of its own in that sense.)